core/build.gradle

```diff
@@ -7,6 +7,20 @@ dependencies {
     implementation group: 'com.google.guava', name: 'guava', version: '30.1-jre'
 }
 
+repositories {
```

Comment:
Can this and the other place be unified in `buildSrc\src\main\groovy\com.chopshop166.chopshoplib.java-common-conventions.gradle`?